packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/location.dart

```diff
@@ -14,13 +14,16 @@ class LatLng {
   /// The latitude is clamped to the inclusive interval from -90.0 to +90.0.
   ///
   /// The longitude is normalized to the half-open interval from -180.0
-  /// (inclusive) to +180.0 (exclusive)
+  /// (inclusive) to +180.0 (exclusive). Avoids normalization if possible to
+  /// avoid unnecessary loss of precision
```

Comment:
The fact that no operations are performed on it if it's already in range is purely an implementation detail, so doesn't belong in the documentation comment. You can move this down to be a normal comment on line 24 instead, to explain the conditional.

Reply:
Good point. I like that reasoning. I'll update it